src/Compiler/Service/TransparentCompiler.fs

```diff
@@ -1514,14 +1542,16 @@ type internal TransparentCompiler
                     let! loadClosure =
                         ComputeScriptClosure
                             fileName
-                            file.Source
+                            (SourceTextNew.ofISourceText file.Source)
```

Comment:
Can you please change the type to `ISourceTextNew` here instead? 
https://github.com/dotnet/fsharp/blob/096ac344bb2a1642cd12778b30b26cb6c3363d4a/src/Compiler/Service/FSharpProjectSnapshot.fs#L126

It should already be getting it, I probably just forgot to update it when I added the new type and it never came up anywhere.

Reply:
Yup!